packages/manager/apps/telecom/src/app/telecom/telephony/line/softphone/softphone.html

```diff
@@ -0,0 +1,23 @@
+<oui-tile
+    data-heading="{{ 'telephony_line_softphone_download_links_title' | translate }}"
+>
+    <oui-tile-definition
+        data-term="{{ 'telephony_line_softphone_download_links_description_label' | translate }}"
+    >
+    </oui-tile-definition>
+</oui-tile>
+<oui-tile
+    class="mt-3"
+    data-heading="{{ 'telephony_line_softphone_configure_title' | translate }}"
+>
+    <oui-tile-description
+        ><span
+            data-translate="telephony_line_softphone_configure_description"
+        ></span
+    ></oui-tile-description>
+</oui-tile>
+<oui-tile
+    class="mt-3"
+    data-heading="{{ 'telephony_line_softphone_theme_title' | translate }}"
+>
+</oui-tile>
```

Comment:
no content for this tile ? What's the purpose then ?

Reply:
It's a nusthell ;)